### PR TITLE
CAL-24 Added Web Console Branding plugins

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Adds branding to the Alliance Distribution
#### Who is reviewing it?
@jlcsmith 
@kcwire 
#### How should this be tested?
Build Alliance, install, observe branding
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-24
#### Screenshots (if appropriate)
https://codice.atlassian.net/browse/CAL-24
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/alliance/pom.xml [hunk 1]
```diff
@@ -135,6 +137,16 @@
             <classifier>features</classifier>
             <type>xml</type>
         </dependency>
+        <dependency>
+            <groupId>org.codice.alliance</groupId>
```
Comment (kcwire) [inline distribution/alliance/pom.xml]: These groupId's do not match the actual artifacts. Need to add `.distribution`.

File: distribution/webconsole-branding-plugin/src/main/java/org/codice/alliance/distribution/webconsole/branding/AllianceBrandingPlugin.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        IOUtils.closeQuietly(ins);
+        brandName = props.getProperty("webconsole.brand.name", "Alliance Web Console");
+        productName = props.getProperty("webconsole.product.name", "Alliance");
+        productURL = props.getProperty("webconsole.product.url", "http://felix.apache.org");
+        productImage = props.getProperty("webconsole.product.image", "/res/alliance/alliance.png");
+        vendorName = props.getProperty("webconsole.vendor.name", "Codice");
+        vendorURL = props.getProperty("webconsole.vendor.url", "http://www.codice.org");
+        vendorImage = props.getProperty("webconsole.vendor.image", "/res/alliance/alliance.png");
+        favIcon = props.getProperty("webconsole.favicon", "/res/alliance/favicon.ico");
+        mainStyleSheet = props.getProperty("webconsole.stylesheet", "/res/ddf/webconsole.css");
```
Comment (kcwire) [inline distribution/webconsole-branding-plugin/src/main/java/org/codice/alliance/distribution/webconsole/branding/AllianceBrandingPlugin.java]: This should be `/res/alliance/webconsole.css` - that should make the felix console happy.

File: distribution/alliance/pom.xml (hunk 1 above)
Comment (bdeining) [inline distribution/alliance/pom.xml]: Fixed

File: distribution/webconsole-branding-plugin/src/main/java/org/codice/alliance/distribution/webconsole/branding/AllianceBrandingPlugin.java (hunk 2 above)
Comment (bdeining) [inline distribution/webconsole-branding-plugin/src/main/java/org/codice/alliance/distribution/webconsole/branding/AllianceBrandingPlugin.java]: Fixed

**Pull request merged**